tsconfig.json

```diff
@@ -19,8 +19,7 @@
       "~tailwind/*": ["./packages/tailwind/src/*"],
       "~tests": ["./packages/components/__tests__/index"],
       "~tests/*": ["./packages/components/__tests__/*"]
-    },
-    "types": ["@testing-library/jest-dom"]
```

Comment:
Looks like the library is still around. We definitely don't need this here?

Reply:
It's only relevant in the tsconfig for writing tests, but since none of the packages extend off base any more, having it here doesn't make a diff. The package still exists in root for the `vitest.setup` config